bionty/base/sources.yaml

```diff
@@ -119,6 +119,13 @@ CellMarker:
     name: CellMarker
     website: http://bio-bigdata.hrbmu.edu.cn/CellMarker
 CellLine:
+  depmap:
+    all:
+      2024-Q2:
+        url: s3://bionty-assets/df_all__depmap__2024-Q2__CellLine.parquet
+        md5: ""
+    name: Dependency Map
+    website: https://depmap.org/portal/
   clo:
```

Comment:
So the order of sources matters as the top one is the default. In this case, do we want users by default use depmap or rather clo? If latter, we should put the clo on top.